frontend/src/pages/article/components/article-form/article-form.tsx

```diff
@@ -59,6 +72,53 @@ const ArticleForm: React.FC<Properties> = ({ articleForUpdate }) => {
     typeof ArticleSubmitType
   > | null>(null);
 
+  const handleInitialData = useCallback(() => {
+    void dispatch(articlesActions.setArticleFormDataFromLocalStorage());
+  }, [dispatch]);
+
+  useEffect(() => {
+    if (!articleForUpdate) {
+      handleInitialData();
+    }
+  }, [handleInitialData, articleForUpdate]);
+
+  useEffect(() => {
+    ((): void => {
+      if (articleForUpdate) {
+        setInitialText(articleForUpdate.text);
+      } else {
+        const { title, text } = articleDataFromLocalStorage ?? {};
+
+        if (title || text) {
+          setIsContentFromLocalStorage(true);
+        }
+
+        setInitialText(text ?? '');
```

Comment:
Not sure why we need this 🥲 as value comes on field from react-hook-form controller

Reply:
I added it because there is a rewriting of an initial value in the text-editor:

  ```
useEffect(() => {
    if (!wasEdited && editor) {
      editor.commands.setContent(initialValue);
    }
  }, [wasEdited, initialValue, editor]);
```